content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -7,29 +7,30 @@ weight: 30
 ---
 
 {{% capture overview %}}
-This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this instance, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
+This tutorial shows you how to develop a native cloud [Cassandra](http://cassandra.apache.org/) deployment on Kubernetes. In this example, a custom Cassandra `SeedProvider` enables Cassandra to discover new Cassandra nodes as they join the cluster.
 
-Deploying stateful distributed applications, like Cassandra, within a clustered environment can be challenging. StatefulSets greatly simplify this process. Please read about [StatefulSets](/docs/concepts/workloads/controllers/statefulset/)  for more information about the features used in this tutorial.
+It can be challenging to deploy stateful distributed applications within a clustered environment. StatefulSets greatly simplify this process. See the [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) documentation for more information on the features used in this tutorial.
```

Comment:
Use [simple and direct language](https://kubernetes.io/docs/home/contribute/style-guide/#content-best-practices). In this case, "It can be challenging to deploy" can be tightened into more active language, and the two sentences can be combined. Also, I'd like us to standardize link prose toward, "For more information, see `<x>`."
```
StatefulSets make it easier to deploy stateful applications within a clustered environment. For more information on the features used in this tutorial, see the [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) documentation.
```
So:
> StatefulSets make it easier to deploy stateful applications within a clustered environment. For more information on the features used in this tutorial, see the [StatefulSet](/docs/concepts/workloads/controllers/statefulset/) documentation.